src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.VisualBasic.xaml

```diff
@@ -416,7 +416,10 @@
                   DisplayName="Custom constants"
                   HelpUrl="https://go.microsoft.com/fwlink/?linkid=2196264"
                   Category="Advanced">
-    <StringProperty.Description>Enter any custom constants for your application in this text box. Entries should be delimited by commas, using this form: Name1="Value1",Name2="Value2",Name3="Value3".</StringProperty.Description>
+    <StringProperty.Description>Use this editor to modify custom constants for your application.</StringProperty.Description>
```

Comment:
Could be made grammatically more consistent with other descriptions via something like:

```suggestion
    <StringProperty.Description>Specifies custom conditional compilation constants for your application.</StringProperty.Description>
```

https://learn.microsoft.com/en-us/dotnet/visual-basic/programming-guide/program-structure/conditional-compilation